src/server/services/wms/qgswmsgetcapabilities.cpp

```diff
@@ -1073,6 +1073,12 @@ namespace QgsWms
                   // or the CRS extent otherwise
                   extent = vl->crs().bounds();
                 }
+                // If CRS is different transform it to layer's CRS
+                else if ( vl->crs() != project->crs() )
+                {
+                  QgsCoordinateTransform ct( project->crs(), vl->crs(), project->transformContext() );
+                  extent = ct.transform( extent );
```

Comment:
Do you need to catch cs exceptions here?

Reply:
yes, but I'm in doubt about what to do in that case: abort and return an exception to the client?

Reply:
... or just log and skip the offending layer?

Reply:
+1 for logging, I think you did well!